Source/SwiftLintFramework/Rules/RuleConfigurations/TestCaseAccessibilityConfiguration.swift

```diff
@@ -1,10 +1,10 @@
 public struct TestCaseAccessibilityConfiguration: RuleConfiguration, Equatable {
     public private(set) var severityConfiguration = SeverityConfiguration(.warning)
-    public private(set) var methodPrefixes: Set<String> = []
+    public private(set) var allowedPrefixes: Set<String> = []
 
     public var consoleDescription: String {
         return severityConfiguration.consoleDescription +
-        	", method_prefixes: [\(methodPrefixes)]"
+            ", allowed_prefixes: [\(allowedPrefixes)]"
```

Comment:
Examples can now specify configurations, so it'd be nice to test something like `disabled_test` as an allowed prefix. https://github.com/realm/SwiftLint/blob/da408b5901c0354a2930e9faf28dd678990fbca3/Source/SwiftLintFramework/Models/Example.swift#L6-L16

Reply:
Is there an example of that working? I tried that and I don't believe it's passed all the way through the tests

Reply:
AFAICT it only is for analyze tests

Reply:
Oh I think you're right. Let's hold off for now, but it should be easy to make normal lint tests apply this configuration value.